chapters/intro-computer-architecture/c-goto-statements/reading/README.md

```diff
@@ -0,0 +1,149 @@
+# C basics: GOTOs
+
+A less addressed concept in C tutorials is the goto statement.
+Using the goto statement, a program can jump to intermediate points within a function.
+These intermediate points are called labels.
+
+## Syntax
+
+Syntax-wise, a label consists of a name followed by the character `:`.
+
+Code example:
+
+ ```c
+#include <stdio.h>
+
+int main()
+{
+    int i, j, k;
+    /* some code */
+do_some_work:
+    /* some other code */
+    work();
+    if (any_work())
+        goto do_some_work;
+    /* some code */
+    return 0;
+}
+```
+
+The program executes a job through the `work()` function.
+In case there are other unfinished jobs, the program execution jumps to the label do_some_work.
+do_some_work marks the point in the program where the processing of a new job begins.
```

Comment:
```suggestion
In case there are other unfinished jobs, the program execution jumps to the label `do_some_work`.
`do_some_work` marks the point in the program where the processing of a new job begins.
```